main.c

```diff
@@ -349,8 +350,10 @@ int main(int argc, char *argv[])
 	if (server == NULL)
 		server = xstrdup(cmus_socket_path);
 
-	if (!remote_connect(server))
+	if (!remote_connect(server)){
+		free(server);
```

Comment:
`server` can also come from argv, in that case this is actually wrong. Since not freeing stuff before returning from `main()` is fine, no need to change anything here.